pcu/PCU.cc

```diff
@@ -207,7 +225,14 @@ void PCU::DebugPrint(const char *format, va_list args) noexcept {
   vfprintf(msg_->file, format, args);
   fflush(msg_->file);
 }
-PCU::PCU(MPI_Comm comm) {
+PCU::PCU() :
+#ifndef SCOREC_NO_MPI
+PCU(MPI_COMM_WORLD)
```

Comment:
You should never actually use `MPI_COMM_WORLD`, instead call `MPI_Comm_dup` on the world communicator.